portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,35 @@
 
 package com.google.sps.servlets;
 
+import com.google.sps.data.Comments;
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Arrays;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Erin!</h1>");
-  }
+    private List<String> comments = new ArrayList<>(Arrays.asList("hello", "world", "goodbye", "earth"));
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+        String json = convertToJson(comments);
+
+        response.setContentType("application/json;");
+        response.getWriter().println(json);
+    }
+
+    private String convertToJson(List jsonToParse) {
+        Gson gson = new Gson();
```

Comment:
Seems this can be moved out and declare as static final

Reply:
What do you mean by moved out?

Reply:
You don't need to create a new instance every time the function is called. you should probably make it an instance variable.

Reply:
gotcha

Reply:
can you address this?

Reply:
Im confused, I pulled it out, is it not showing the update?

Reply:
doesn't look like it's showing up
https://github.com/erinlimbo/Erin-STEP/blob/json/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java#L47-L51

Reply:
Ok then maybe I dont quite understand where I am suppose to put it. It shows it as how I placed it, so where exactly am I suppose to keep the function? I cant pull it into anything outside of DataServlet class?